test/test_loggers.jl

```diff
@@ -167,20 +167,29 @@ end
     runname = "run-$(UUIDs.uuid4())"
     r = createrun(mlf, e.experiment_id)
 
+    tempfilename = "./mlflowclient-tempfile.txt"
+
+    open(tempfilename, "w") do file
+        write(file, "Hello, world!\n")
+    end
+
```

Comment:
I think it is more usual to use the functions `tempname` or `tempdir` to create temporary files during CI, although I think this still works. But perhaps you have reasons for doing it this way?